### ISSUE TITLE
`choco info` Command Doesn't Show Package URL

### ISSUE THREAD

**Issue by FranklinYu** [chocolatey/choco#1728]
### How Did You Get This To Happen? (Steps to Reproduce)
`choco info nodejs.install`, or essentially any package

### What You Are Seeing?
```text
 Package url
```

### What is Expected?
```text
 Package url: https://chocolatey.org/packages/nodejs.install
```

### Output Log

<details>
<summary>Full Log Output</summary>

<p>

~~~sh
2019-02-11 11:51:13,716 9948 [DEBUG] - XmlConfiguration is now operational
2019-02-11 11:51:13,749 9948 [INFO ] - ============================================================
2019-02-11 11:51:13,859 9948 [DEBUG] - User may be subject to UAC, checking for a split token (not 100% 
 effective).
2019-02-11 11:51:13,877 9948 [INFO ] - Chocolatey v0.10.11
2019-02-11 11:51:13,884 9948 [DEBUG] - Chocolatey is running on Windows v 10.0.17134.0
2019-02-11 11:51:13,886 9948 [DEBUG] - Attempting to delete file "C:/ProgramData/chocolatey/choco.exe.old".
2019-02-11 11:51:13,887 9948 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\choco.exe.old".
2019-02-11 11:51:13,896 9948 [DEBUG] - Command line: "C:\ProgramData\chocolatey\choco.exe" info nodejs.install
2019-02-11 11:51:13,897 9948 [DEBUG] - Received arguments: info nodejs.install
2019-02-11 11:51:13,931 9948 [DEBUG] - RemovePendingPackagesTask is now ready and waiting for PreRunMessage.
2019-02-11 11:51:13,936 9948 [DEBUG] - Sending message 'PreRunMessage' out if there are subscribers...
2019-02-11 11:51:13,942 9948 [DEBUG] - [Pending] Removing all pending packages that should not be considered installed...
2019-02-11 11:51:14,095 9948 [DEBUG] - The source 'https://chocolatey.org/api/v2/' evaluated to a 'normal' source type
2019-02-11 11:51:14,097 9948 [DEBUG] - 
NOTE: Hiding sensitive configuration data! Please double and triple 
 check to be sure no sensitive data is shown, especially if copying 
 output to a gist for review.
2019-02-11 11:51:14,103 9948 [DEBUG] - Configuration: CommandName='info'|
CacheLocation='C:\Users\FranklinYu\AppData\Local\Temp\chocolatey'|
ContainsLegacyPackageInstalls='True'|
CommandExecutionTimeoutSeconds='2700'|WebRequestTimeoutSeconds='30'|
Sources='https://chocolatey.org/api/v2/'|

SourceType='normal'|Debug='False'|Verbose='True'|Trace='False'|
Force='False'|Noop='False'|HelpRequested='False'|
UnsuccessfulParsing='False'|RegularOutput='True'|QuietOutput='False'|
PromptForConfirmation='False'|AcceptLicense='False'|
AllowUnofficialBuild='False'|Input='nodejs.install'|
AllVersions='False'|
SkipPackageInstallProvider='False'|Prerelease='False'|ForceX86='False'|
OverrideArguments='False'|NotSilent='False'|
ApplyPackageParametersToDependencies='False'|
ApplyInstallArgumentsToDependencies='False'|IgnoreDependencies='False'|
AllowMultipleVersions='False'|AllowDowngrade='False'|
ForceDependencies='False'|Information.PlatformType='Windows'|
Information.PlatformVersion='10.0.17134.0'|
Information.PlatformName='Windows 10'|
Information.ChocolateyVersion='0.10.11.0'|
Information.ChocolateyProductVersion='0.10.11'|
Information.FullName='choco, Version=0.10.11.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb'|

Information.Is64BitOperatingSystem='True'|
Information.Is64BitProcess='True'|Information.IsInteractive='True'|
Information.UserName='FranklinYu'|Information.UserDomainName='internal'|
Information.IsUserAdministrator='True'|
Information.IsUserSystemAccount='False'|
Information.IsUserRemoteDesktop='False'|
Information.IsUserRemote='False'|Information.IsProcessElevated='False'|
Information.IsLicensedVersion='False'|Information.LicenseType='Foss'|
Features.AutoUninstaller='True'|Features.ChecksumFiles='True'|
Features.AllowEmptyChecksums='False'|
Features.AllowEmptyChecksumsSecure='True'|
Features.FailOnAutoUninstaller='False'|
Features.FailOnStandardError='False'|Features.UsePowerShellHost='True'|
Features.LogEnvironmentValues='False'|Features.LogWithoutColor='False'|
Features.VirusCheck='False'|
Features.FailOnInvalidOrMissingLicense='False'|
Features.IgnoreInvalidOptionsSwitches='False'|
Features.UsePackageExitCodes='True'|
Features.UseFipsCompliantChecksums='False'|
Features.ShowNonElevatedWarnings='True'|
Features.ShowDownloadProgress='True'|
Features.StopOnFirstPackageFailure='False'|
Features.UseRememberedArgumentsForUpgrades='False'|
Features.IgnoreUnfoundPackagesOnUpgradeOutdated='False'|
Features.RemovePackageInformationOnUninstall='False'|
Features.ScriptsCheckLastExitCode='False'|
ListCommand.LocalOnly='False'|
ListCommand.IdOnly='False'|ListCommand.IncludeRegistryPrograms='False'|
ListCommand.PageSize='25'|ListCommand.Exact='True'|
ListCommand.ByIdOnly='False'|ListCommand.ByTagOnly='False'|
ListCommand.IdStartsWith='False'|ListCommand.OrderByPopularity='False'|
ListCommand.ApprovedOnly='False'|
ListCommand.DownloadCacheAvailable='False'|
ListCommand.NotBroken='False'|
ListCommand.IncludeVersionOverrides='False'|
UpgradeCommand.FailOnUnfound='False'|
UpgradeCommand.FailOnNotInstalled='False'|
UpgradeCommand.NotifyOnlyAvailableUpgrades='False'|
UpgradeCommand.ExcludePrerelease='False'|
NewCommand.AutomaticPackage='False'|
NewCommand.UseOriginalTemplate='False'|SourceCommand.Command='unknown'|
SourceCommand.Priority='0'|SourceCommand.BypassProxy='False'|
SourceCommand.AllowSelfService='False'|
SourceCommand.VisibleToAdminsOnly='False'|
FeatureCommand.Command='unknown'|ConfigCommand.Command='unknown'|
PinCommand.Command='unknown'|OutdatedCommand.IgnorePinned='False'|
Proxy.BypassOnLocal='True'|
2019-02-11 11:51:14,105 9948 [DEBUG] - _ Chocolatey:ChocolateyInfoCommand - Normal Run Mode _
2019-02-11 11:51:14,108 9948 [DEBUG] - Searching for package information
2019-02-11 11:51:14,113 9948 [DEBUG] - Running list with the following filter = 'nodejs.install'
2019-02-11 11:51:14,114 9948 [DEBUG] - --- Start of List ---
2019-02-11 11:51:15,111 9948 [INFO ] - nodejs.install 11.9.0 [Approved]
2019-02-11 11:51:15,120 9948 [INFO ] -  Title: Node JS (Install) | Published: 1/31/2019
 Package approved as a trusted package on Jan 31 2019 05:58:54.
 Package testing status: Passing on Jan 31 2019 00:44:27.
 Number of Downloads: 2430429 | Downloads for this version: 17280
 Package url
 Chocolatey Package Source: https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/nodejs.install
 Package Checksum: 'HkEYEQCNQOsXpFjApZxAAT+LHK53RYBDPHRbp32COazk6Ft1feFNWoqXhEHm8tB19xN5DK/LMMSerB47qN536g==' (SHA512)
 Tags: nodejs node javascript npm admin foss cross-platform
 Software Site: http://nodejs.org/
 Software License: https://github.com/nodejs/node/blob/master/LICENSE
 Software Source: https://github.com/nodejs/node
 Documentation: https://nodejs.org/en/docs/
 Issues: https://github.com/nodejs/node/issues
 Summary: Node JS - Evented I/O for v8 JavaScript.
 Description: Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.
  
  This package runs the official Node JS installer, resulting in Node.exe and NPM being installed under Program Files, and added to the SYSTEM PATH environment variable.
  
  ## Notes
  While this package now provides both **Current** and **LTS** releases of Node.js, it is still recommended to use the [nodejs-lts][] package if only targeting the latest LTS release is required.
  
  [nodejs-lts]: https://chocolatey.org/packages/nodejs-lts "Node.js LTS Package"

2019-02-11 11:51:15,135 9948 [DEBUG] - --- End of List ---
2019-02-11 11:51:15,136 9948 [WARN ] - 1 packages found.
2019-02-11 11:51:15,139 9948 [DEBUG] - Sending message 'PostRunMessage' out if there are subscribers...
2019-02-11 11:51:15,142 9948 [DEBUG] - Exiting with 0
~~~

</p>

</details>

**Comment by FranklinYu** [comment]
Corresponding source:

https://github.com/chocolatey/choco/blob/0c720d6f7637acabe5341bff9fc94ced0bc6328e/src/chocolatey/infrastructure.app/services/NugetService.cs#L167-L169

**Comment by TheCakeIsNaOH** [comment]
The `PackageSourceUrl` is exposed `IPackageSearchMetadata` by the newer NuGet SDK libraries, so this should be able to be easily fixed as a part of or after #508

**Comment by TheCakeIsNaOH** [comment]
This was fixed in v2.0.0 with the new NuGet libraries. 
![image](https://github.com/chocolatey/choco/assets/38865330/470d9d6a-7ae7-43a4-ba99-8674b4f7efaa)

I'm just not sure if there are tests for this, or even if tests can be done at all, since there is only something that shows up for some http based repositories (like CCR).

**Comment by corbob** [comment]
I can confirm that the package url and package source only show up on CCR, and not on Nexus repositories. As we don't generally test against CCR, I would say this can be closed and attached to the 2.0.0 milestone.

**Comment by corbob** [comment]
Note: I am purposely leaving this issue open at the moment, as it's waiting on chocolatey/docs#781 to be merged, and then the GitHub release will also be updated.
